zeppelin-web-angular/src/app/visualizations/table/table-visualization.component.html

```diff
@@ -37,12 +37,13 @@
 <nz-table #table
           nzSize="small"
           nzShowSizeChanger
+          [nzScroll]="{ x: '1300px' }"
           [nzData]="rows"
           [nzFooter]="aggregatesFooter">
   <thead>
   <tr>
     <th *ngFor="let col of columns"
-        nzWidth="800px"
+        nzWidth="200px"
```

Comment:
I'm just curious. is the value restricted to a minimum size? Otherwise, how does this setting affect to UI?

Reply:
@jongyoul ![image](https://github.com/apache/zeppelin/assets/23468656/f229b123-79c1-4535-98f4-d68fda642d54)
Yes, this setting will restrict its minimum value.